app/command.go

```diff
@@ -479,13 +479,7 @@ func (a *App) GetCommand(commandId string) (*model.Command, *model.AppError) {
 		return nil, model.NewAppError("GetCommand", "api.command.disabled.app_error", nil, "", http.StatusNotImplemented)
 	}
 
-	result := <-a.Srv.Store.Command().Get(commandId)
-	if result.Err != nil {
-		result.Err.StatusCode = http.StatusNotFound
-		return nil, result.Err
-	}
-
-	return result.Data.(*model.Command), nil
+	return a.Srv.Store.Command().Get(commandId)
```

Comment:
In this case is not exactly the same, In the previous implementation the `StatusCode` was changed to `StatusNotFound` for any error, and the `Get` method returns an `InternalServerError` StatusCode, so here we can't simplify it this way. I think we can rethink what must return `Get` method in the future, but for now we should stick to the current implementation because this serie of PRs are for other purpose.

Reply:
yeah sorry! didn't notice the `StatusCode` change. will update it.